pom.xml

```diff
@@ -23,8 +23,8 @@
 
   <properties>
     <pac4j.version>4.3.0</pac4j.version>
-    <dropwizard-pac4j.version>4.0.0</dropwizard-pac4j.version>
-    <java.version>1.8</java.version>
+    <dropwizard-pac4j.version>5.0.0-SNAPSHOT</dropwizard-pac4j.version>
+    <java.version>11</java.version>
```

Comment:
I doubt this really works as you didn't change the `maven-compiler-plugin`:

```xml
      <plugin>
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-compiler-plugin</artifactId>
        <version>3.8.1</version>
        <configuration>
          <source>${java.version}</source>
          <target>${java.version}</target>
        </configuration>
      </plugin>
```

It should be:

```xml
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-compiler-plugin</artifactId>
                <version>3.8.1</version>
                <configuration>
                    <release>${java.version}</release>
                    <encoding>UTF-8</encoding>
                </configuration>
            </plugin>
```
```

Reply:
it does run locally, once you cut 5.0.0 i can go ahead and update this so you can verify for yourself otherwise im happy to screen share send output or whatever you need to verify this works or you can run it locally

Reply:
this is also the same as what was done here 
https://github.com/pac4j/dropwizard-pac4j/blob/master/pom.xml#L92

Reply:
Indeed, this is the same as in the library itself and it works (although I didn't think so).

I trust you, but I prefer to be careful as sometimes, people only submit a PR without really testing it.

And you didn't need the v5.0.0 release, you can test the v5.0.0-SNAPSHOT (available in the Sonatype snapshots repository).

Merging...

Reply:
BTW, I updated the version of the demo to v5.0.0-SNAPSHOT and started it successfully.

Reply:
I just cut the v5.0.0 release of `dropwizard-pac4j`, it will show up in the Maven central repository within a few hours.